README.md

```diff
@@ -67,6 +67,11 @@ Upserts all data types without specifying a schema
 
 ### three_operations
 Shows how to use upsert, update and delete operations
+
+### priority_first_sync_for_high_volume_initial_syncs
```

Comment:
Please get this reviewed by Tech writers.

Reply:
Is it required? There was no review by tech writers in older prs.

Reply:
We should actually have it reviewed by tech writers as well, but if Alison is reviewing, it should be fine.